src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,56 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set 크기 확인")
+    @Test
+    void set_크기_확인() {
+        int expectedSize = 3;
+
+        assertThat(numbers.size()).isEqualTo(expectedSize);
```

Comment:
비교를 잘해주셨어요 👍 
아래와 같이도 할 수 있는 점 참고 부탁드릴께요 😄 
```suggestion
        assertThat(numbers.size()).hasSize(expectedSize)
```

Reply:
피드백 감사합니다!!